hrjobcontract/views/modalForm.html

```diff
@@ -560,33 +560,55 @@ <h4 class="modal-title">{{copy.title}}</h4>
                 </div>
             </uib-tab>
             <uib-tab heading="Leave">
+              <div ng-controller="FormLeaveCtrl">
```

Comment:
This was because I was basing myself on the existing code, which doesn't use the **controller as** syntax